app/controllers/metadata_controller.rb

```diff
@@ -15,6 +15,10 @@ def descriptive
   private
 
   def cocina
-    Repository.find(params[:item_id])
+    if params.key?(:user_version_id)
+      Repository.find_user_version(params[:item_id], params[:user_version_id])
```

Comment:
I'm not sure why this is showing as not covered in tests. Is there something I could change in the spec below?

Reply:
I think you'd need a request spec to cover this change, like https://github.com/sul-dlss/argo/pull/4514/files#diff-723822a1f53de8c8a1789d62ffa3b5c2b773e72b24881c5faf229189f1864a19

Reply:
Thanks, I'll work on that.